project/src/main/webapp/gmail.js

```diff
@@ -110,14 +105,57 @@ function appendPre(message) {
     pre.appendChild(textContent);
 }
 
-var question = "What day is the livestream?";
-var arrayList = [];
-var totalObjects = {};
+function getQuery() {
+    const query = document.getElementById("text-input").value;
+    console.log("This is the textarea: " + query);
+    return query;
+}
+
+
+/**
+ * Get Message with given ID.
+ *
+ * @param  {String} messageId ID of Message to get.
+ */
+function getMessage(messageId) {
+    const messageRequest = gapi.client.gmail.users.messages.get({
+        'userId': 'me',
+        'id': messageId,
+        'format': "raw"
+    });
+    messageRequest.execute(response => {
+        //Convert from base64 encoding to text.
+        const decodedEmail = atob(response.raw.replace(/-/g, '+').replace(/_/g, '/'));
+        var hasFormatFlowed = false;
```

Comment:
Refrain from using var when declaring a variable.

Reply:
I just changed this!